examples/gno.land/r/x/manfred_outfmt/gno.mod

```diff
@@ -1,3 +1,5 @@
+// Draft
```

Comment:
this is actually ignoring the content of the gno.mod file, is that something we want?

Reply:
Yes

This package depends on p/demo/rand, which uses a stdlib which does not exist (`math/rand`)